tools.sh

```diff
@@ -24,6 +24,11 @@ case "${ARCH}" in
         SNC_GENERATE_MACOS_BUNDLE=1
 	SNC_GENERATE_WINDOWS_BUNDLE=1
 	;;
+    aarch64)
+        yq_ARCH="arm64"
+        SNC_GENERATE_MACOS_BUNDLE=1
+	SNC_GENERATE_WINDOWS_BUNDLE=
```

Comment:
Is macOS bundle generation intentionally enabled because you are targetting the M1? 
Feel free to remove the mix of spaces and tabs in this area while you are changing this (should be only spaces I think)

Reply:
correct. the MAC bundles are targeting M1.